routes/user.ts

```diff
@@ -34,8 +34,78 @@ export const createUserRouter = ({ userModel }) => {
      *         description: Internal server error
      */
     userRouter.get('/', userController.getAll)
+    /**
+     * @swagger
+     * /api/v1/users/:id:
+     *   get:
+     *     summary: Get user by id
+     *     description: Retrieves the user.
+     *     security:
+     *       - bearerAuth: []
+     *     responses:
+     *       200:
+     *         description: Successful response
+     *         content:
+     *           application/json:
+     *             schema:
+     *              properties:
+     *                 
+     *                  username:
+     *                      type: string
+     *                  
+     *       400:
+     *         description: Unauthorized
+     *       404:
+     *         description: User not found
+     *       500:
+     *         description: Internal server error
+     */
     userRouter.get('/:id', userController.getById)
+    /**
+ * @swagger
+ * /api/v1/users:
+ *   post:
+ *     summary: Create user
+ *     description: Creates a new user.
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/User'
+ *     responses:
+ *       '201':
+ *         description: Successful response
+ *         content:
+ *           application/json:    
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 username:
+ *                   type: string
+ *                 id:
+ *                   type: string
+ *       '400':
+ *         description: Unauthorized
+ *       '500':
+ *         description: Internal server error
+ * 
+ * components:
+ *   schemas:
+ *     User:
+ *       type: object
+ *       required:
+ *         - username
+ *         - password
+ *       properties:
+ *         id:
+ *           type: integer
+ *         username:
+ *           type: string
+ *         password:
+ *           type: string
+ */
```

Comment:
Why did you comment out this section of the code?

Reply:
It is not a comment, it is the functionality of Swagger, you are free to change sections, so that later when you make your routes you document them and the open api interface can be used

Reply:
I'm so sorry, I never use Swagger, I had no idea about this. 😅